apps/frontend/src/views/Login.vue

```diff
@@ -123,5 +140,9 @@ export default class Login extends Vue {
         SnackbarModule.notify(error.response.data.message);
       });
   }
+
+  loginGithub() {
+    window.location.href = `http://${window.location.hostname}:${process.env.PORT || 3000}/authn/github`;
```

Comment:
Can this be handled using the vue router? `this.$router.push(route)`

Also does this actually need the full window location? `/authn/github` is being served by the backend which should always be on the same host.

Reply:
![image](https://user-images.githubusercontent.com/66680985/102792047-ea88ca80-4375-11eb-83cb-3a55dcdd20f7.png)
It doesn't seem to work on the frontend port, and router [doesn't support](https://stackoverflow.com/questions/44595929/vue2-navigate-to-external-url-with-location-href) visiting the page even if its on the same host but different port

Reply:
Is this only in dev mode that it doesn't work? The frontend should proxy unknown requests through to the backend, also the `http` scheme being hardcoded here is going to cause problems for people using https.

Reply:
Router doesn't work in production either, however, the http scheme is a problem. I've set it now to be relative to current location.